### PR TITLE
Feature/fix translation cache

### PR DESCRIPTION
**Pull request by sergeysenja1992**

### CODE REVIEW

**Review by ishkurko — APPROVED** [review]

File: scripts/replace.build.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+var replace = require('replace-in-file');
```
Comment (Kaskyi) [inline scripts/replace.build.js:1]: const

File: scripts/replace.build.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+var buildTimestamp = new Date().getTime();
```
Comment (Kaskyi) [inline scripts/replace.build.js:2]: const

File: scripts/replace.build.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+const options = {
+    files: ['src/environments/environment.ts', 'src/environments/environment.prod.ts'],
+    from: /buildTimestamp: '\d*'/g,
+    to: `buildTimestamp: '${buildTimestamp}'`,
+    allowEmptyPaths: false,
+};
+
+try {
+    let changedFiles = replace.sync(options);
+    console.log('Build timestamp set: ' + buildTimestamp);
```
Comment (Kaskyi) [inline scripts/replace.build.js:12]: console.info

File: scripts/replace.build.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+var buildTimestamp = new Date().getTime();
+const options = {
+    files: ['src/environments/environment.ts', 'src/environments/environment.prod.ts'],
+    from: /buildTimestamp: '\d*'/g,
+    to: `buildTimestamp: '${buildTimestamp}'`,
+    allowEmptyPaths: false,
+};
+
+try {
+    let changedFiles = replace.sync(options);
```
Comment (Kaskyi) [inline scripts/replace.build.js:11]: const

**Review by Kaskyi — APPROVED** [review]

File: scripts/replace.build.js (hunk 1 above)
Comment (sergeysenja1992) [reply scripts/replace.build.js]: fixed

File: scripts/replace.build.js (hunk 3 above)
Comment (sergeysenja1992) [reply scripts/replace.build.js]: fixed

File: scripts/replace.build.js (hunk 4 above)
Comment (sergeysenja1992) [reply scripts/replace.build.js]: fixed

**Pull request merged**